src/model/AirLoopHVACOutdoorAirSystem_Impl.hpp

```diff
@@ -91,20 +91,23 @@ namespace model {
 
       virtual bool addToNode(Node& node) override;
 
+      // Take a shortcut to avoid also checking each AirLoopHVAC's OutdoorAirSystem's component like HVACComponent logically does
+      virtual boost::optional<AirLoopHVAC> airLoopHVAC() const override;
```

Comment:
Instead of defining an `airLoop()` method, just override the `HVACComponent_Impl::airLoopHVAC()` so we can shortcut